Caboodle/PhoneDialer/PhoneDialer.netstandard.cs

```diff
@@ -2,8 +2,7 @@
 {
     public static partial class PhoneDialer
     {
-        internal static bool IsSupported =>
-            throw new NotImplementedInReferenceAssemblyException();
+        internal static bool IsSupported => false;
```

Comment:
Should we be doing this in all?

Reply:
I was just going out on a limb here and have the IsSupported bits NOT throw ever so that we can write nicer tests. But I don't really know if we will ever do this or how we should handle this idea.